tmpl/live-editor.handlebars

```diff
@@ -17,7 +17,7 @@
 
             <div class="scratchpad-canvas-loading">
                 <img src="{{imagesDir}}/spinner-large.gif">
-                <span class="hide-text">{{#_}}Loading...{{/_}}</span>
+                <span class="hide-text">Loading...</span>
```

Comment:
I removed the {{#_}} calls since they don't actually get handled by webapp (as you recall from my PRs a few weeks ago, and this template in particular gets completely overriden by webapp anyway.

Reply:
Does this mean we can get rid of the handlebars plugin which was handling `{{#_}}`/`{{/_}}`?